.github/workflows/percy-demo-tests.yml

```diff
@@ -0,0 +1,68 @@
+name: Execute percy demo tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      dashboardUrl:
+        description: 'Environment to run tests against'
+        required: true
+        default: 'https://staging-demo.saleor.io/'
+      apiUrl:
+        required: true 
+        description: 'Environment api to run tests against'
+        default: 'https://staging-demo.saleor.io/graphql/'
+
+  schedule:
+    - cron: '00 2 * * 1-5'
+
+  repository_dispatch:
+    types: [demo-tests]
+
+jobs:
+  cypress-run-test-percy:
+    runs-on: ubuntu-latest
+    container: cypress/browsers:node18.12.0-chrome106-ff106
```

Comment:
Based on: https://github.com/cypress-io/cypress-docker-images/tree/master/browsers#naming-scheme

I'd suggest to make versions both for `node` and `chrome` as an inputs, as otherwise it would require manual change each time the version for node or chrome is changed

Reply:
I think I should change it in all of our cypress-related workflows. First we assume that this workflow will be running on schedule and rarely on "workflow-dispatch" so inputs won't help much. I'm thinking maybe I can take node version from ".nvmrc" file, and then send an API request to take latest container for this node version and browser

Reply:
But I would create separate PR for that change for this and other workflows

Reply:
Fair enough 👍 
I'm resolving the conversation then